app/views/includes/scripts.html

```diff
@@ -1,6 +1,4 @@
 <!-- Javascript -->
-<script src="/public/javascripts/jquery-1.11.3.js"></script>
```

Comment:
will users get this change as part of the update process?

Reply:
No